xarray_sentinel/sentinel1.py

```diff
@@ -3,14 +3,28 @@
 import warnings
 
 import numpy as np
+import rioxarray  # type: ignore
 import xarray as xr
 
 from xarray_sentinel import conventions, esa_safe
 from xarray_sentinel.esa_safe import PathType
 
 
-def open_gcp_dataset(product_path: PathType) -> xr.Dataset:
-    geolocation_grid_points = esa_safe.parse_geolocation_grid_points(product_path)
+def read_swath_attributes(annotation_path: PathType) -> T.Dict[str, T.Any]:
+    swath_timing = esa_safe.parse_swath_timing(annotation_path)
+    linesPerBurst = int(swath_timing["linesPerBurst"])
```

Comment:
Are those case needed? Data type should be already correct thanks to xmlschema. If not it is probably a bug in esa_safe.py.

Reply:
Function removed in 6ad0e6b47a8648dc9df7adce1abf58294700b821